uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/settings.js

```diff
@@ -92,6 +92,13 @@ define([
       $log.debug('Service: SettingsService');
 
       return {
+
+        /**
+         * Get app setting form backend using Setting Entity
+         *
+         * @param  {array} fields
+         * @return {array}
```

Comment:
May be you mean `@return {Promise} resolves to array of settings` or just `@return {Promise}`?

Reply:
Right it returns a `promise` which resolves to array. fixed!